### DISCUSSION TITLE
How to get Hardware Information like storage capacity and type, etc?

### DISCUSSION THREAD

**Discussion by ChangeYourself0613** [vercel/next.js#68922] [Help]
### Summary

I built desktop app using electron + next.js.
So I made exe app using electron.
But in that app, I wanna get device's hardware information, but don't know how to do.
Plz feel free to reach out to me if you have any experiences.
Thanks

**Comment by Juneezee** [comment]
Have you tried the `os` module? https://nodejs.org/api/os.html

**Reply by ChangeYourself0613** [reply]
Thanks for your kind of reply.
could you provide me example code?

**Reply by Juneezee** [reply]
There are many examples on the web. Here is one of the gist that I found on Google: https://gist.github.com/jh3y/e25e2b15cff6b63f4f425fa4787d26fd
